src/runtime/GlobalObjectBuiltinRegExp.cpp

```diff
@@ -640,37 +646,50 @@ static Value builtinRegExpUnicodeGetter(ExecutionState& state, Value thisValue,
 {
     return builtinRegExpOptionGetterHelper(state, thisValue, RegExpObject::Option::Unicode);
 }
+#define DEFINE_GETTER_LEGACY_REGEXP_PROPERTY(stringView)                                                          \
+    if (!thisValue.isPointerValue() || thisValue.asPointerValue() != state.context()->globalObject()->regexp()) { \
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "thisValue is not RegExp");                 \
+    }                                                                                                             \
+    if (stringView.isEmpty()) {                                                                                   \
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "Value is empty");                          \
```

Comment:
For error messages, how about exploiting existing messages defined in `ErrorObject::Messages`?
e.g) `ErrorObject::Messages::GlobalObject_ThisNotRegExpObject` for the first error